plugins/powerd/fu-plugin-powerd.c

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2021 Twain Byrnes <binarynewts@google.com>
+ * Copyright (C) 2021 George Popoola <gpopoola@google.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <fwupdplugin.h>
+
+#define MINIMUM_BATTERY_PERCENTAGE_FALLBACK 70
+
+struct FuPluginData {
+        GDBusProxy    *proxy;   /* nullable */
```

Comment:
don't you need to unref this in destroy?

Reply:
The fu_plugin_destoy () function was added in the latest upload.